src/common/workspace-store.ts

```diff
@@ -138,6 +138,7 @@ export class WorkspaceStore extends BaseStore<WorkspaceStoreModel> {
     super({
       configName: "lens-workspace-store",
     });
+    this.workspaces.get(WorkspaceStore.defaultId).enabled = true;
```

Comment:
Do you mean, "on every boot"?

Reply:
I think this only happens the first time after Lens install

Reply:
No, it happens every time Lens is full quit (meaning including the tray icon is gone). A better solution would be to have the default workspace default to true.

Reply:
By this, I mean the bug

Reply:
Yes I see, (and I don't remember if a user is currently allowed to disable the default workspace).